changelogs/fragments/794-s3_lifecycle_abort_expire.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+- Add I(abort_incomplete_multipart_upload_days) and I(expire_object_delete_marker) fields to I(s3_lifecycle)
```

Comment:
```suggestion
- s3_lifecycle - Add ``abort_incomplete_multipart_upload_days`` and ``expire_object_delete_marker`` parameters (https://github.com/ansible-collections/community.aws/pull/794).
```

Reply:
@alinabuzachis why do you use double backticks here but `I()` elsewhere?